Update changelog for v0.9.15

Amazinite opened the pull request:
Updates the changelog with everything up to 4ba17f2.

BlazingDiesel commented on changelog:
```diff
@@ -149,6 +267,13 @@ Version 0.9.15:
+      * Active missions that you no longer have space for now count as aborted instead of simply failed. (@Amazinite)
+      * Fighters and drones now retreat and redeploy at 50% and 75% health respectively instead of 25% and 35%, increasing their survivability. (@Amazinite)
+      * Ships which are appeasing no longer drop their cargo if there are no hostile ships to drop it for. (@DeBlister)
+      * The inaccuracy of a projectile si now also applied to the firing effect. (@DeBlister)
```
```suggestion
      * The inaccuracy of a projectile is now also applied to the firing effect. (@DeBlister)
```
A teeny little typo.

petervdmeer commented on changelog:
```diff
@@ -217,11 +354,22 @@ Version 0.9.15:
+    * Fixed broken --talk launch flag behavior. (@samrocketman)
+    * Added an Average method to WeightedLists to compute the average of its contents. (@tehhowch)
+    * The correct folder for the AppStream file is now used. (@quyykk)
+    * All unit test and integration test files are now located in dedicated files. (@petervdmeer)
```
```suggestion
    * All unit test and integration test files are now located in dedicated directories. (@petervdmeer)
```

EjoThims commented on changelog:
```diff
@@ -83,23 +115,94 @@ Version 0.9.15:
+      * Change the government of some hostile mission NPCs to dummy governments to prevent permanent reputation loss with the main government. (@PeacefulPotato)
+      * Hai jobs now grant a small amount of reputation upon completion, allowing access to Hai-home without the need to fight the Unfettered. (@Terin)
+      * Moved the defer option in "There Might Be Riots 1" to an earlier point so that information isn't repeated if the player defers multiple times. (@Anarchist2)
+      * Unwanted Cargo can no trigger anywhere outside of Hai space instead of only in human space. (@Anarchist2)
```
```suggestion
      * Unwanted Cargo can now trigger anywhere outside of Hai space instead of only in human space. (@Anarchist2)
```

Amazinite commented:
Finally up to date (although I should also probably update the credits file in this PR). Any candidates for stuff that should be moved to "big changes"? A group of similar changes could all be moved as well. For example, the line under "big changes" talking about the Korath additions encompasses multiple PRs.

Speaking of which, #6891 mentioned quite a few names as being involved in KEE, but I've only listed the first four names and didn't include anyone listed under "additional help." Is there anyone else that should be listed? Should everyone be listed? I don't think we've had any massive collaborative projects in the past with this many people involved, so I suppose we could set the precedent now for who gets the shoutout in the changelog for such projects.

quyykk commented:
Some candidates:

- the engine rebalance
- parallax
- coalition ships buff

Zitchas commented:
I don't think the coalition ships buff was that big. Engine rebalance and Parallax, on the other hand, were major things that will impact everyone.

As for credits for big cooperative projects, I'd be inclined to list everyone. It's the first big such project, lets showcase the fact that it was a team effort with a lot of people on the team.

ravenshining commented:
If you're going to expand the Korath credit, add Warp-Core - they're not in the top post because they didn't really get involved before it was PR'd, but once the PR hit they gave a massive amount of help with stats, and providing valuable contrary opinions against which to better our work.

Amazinite commented:
> the engine rebalance

I don't think the engine rebalance by itself is all that big, but I think that we could mention the variety of balance changes made and say "refer to the balance section for the specifics."

> coalition ships buff

Ditto.

> parallax

Parallax doesn't really seem like that big a change to me, although I do know that it was mentioned on the last changelog update PR when I asked.

ravenshining commented:
Oh, I was breathtaken by seeing parallax for the first time when I was summoned back here.  Definitely worth a mention!

Amazinite commented:
I've moved the specifics of the updates to star graphics down from big changes, and added a big changes line about all the graphical improvements, with the addition of parallax being mentioned.

UnciaPrima commented:
Hi there,
There's a typo on line 217 :)

"Most dedicated battery outfits have recieved a 4x buff." -> RECEIVED

Anarchist2 commented on changelog:
```diff
@@ -14,6 +18,27 @@ Version 0.9.15:
+      * The Coalition Finisher now plays a sound effect when it impacts a target. (@Arachi-Lover)
+      * The Pre-Hai first contact mission for the Wanderers no longer offers if you've met the Unfettered, but not the Hai. (@Arachi-Lover)
+      * Fixed errors in various ship images. (@leklachu)
+      * Fixed the lighting on a new Korath ship sprite. (@Anarchist2)
```
Should we list bug fixes for content added in the same update? From a holistic (that's definitely not the right word) viewpoint, there isn't any difference between a bug being noticed and fixed when reviewing a PR and a bug being fixed in a follow up PR.
